docs/description_cheat_sheet.rst

```diff
@@ -1,5 +1,5 @@
 ***********************
-Description Cheat Sheet
+Cheat Sheet
 ***********************
 
 dist_descriptions
```

Comment:
According to our code style PEP8, please capitalize the first word in all comments --> "General" and "Example".

For "semantics" you could add these two comments comments too.